bot/exts/code_jams/_cog.py

```diff
@@ -176,6 +176,16 @@ async def move(self, ctx: commands.Context, member: Member, *, new_team_name: st
             view=JamConfirmation(author=ctx.author, callback=callback)
         )
 
+    @codejam.command()
+    @commands.has_any_role(Roles.admins, Roles.events_lead)
+    async def add(self, ctx: commands.Context, member: Member, is_leader: bool = False, *, team_name: str) -> None:
```

Comment:
If you are changing other things, maybe consider making this `Optional[bool]`? This enables a special behaviour, where if it fails to match the argument it will skip it - and go to the next. I believe this is what you were looking to have happen with the default?

See the [documentation about it](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html?highlight=optional#typing-optional).